### PR TITLE
test against tagged DiffBase versions instead of DiffBase master

### PR DESCRIPTION
**Pull request by jrevels**
ref @tkelman's [comment here](https://github.com/JuliaDiff/ForwardDiff.jl/pull/179#discussion_r97707014)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ script:
         julia -e 'Pkg.clone(pwd()); Pkg.build("ForwardDiff"); Pkg.checkout("DiffBase"); Pkg.test("ForwardDiff"; coverage=true)';
```
Comment (tkelman) [inline .travis.yml]: here as well?

Comment (jrevels) [inline .travis.yml]: yup, thanks!

**Comment by tkelman** [comment]
Can you please create (then enable branch protection for) a release-0.3 branch starting from this https://github.com/JuliaDiff/ForwardDiff.jl/commit/791cd934166d6dd03705a79d2f346802268636ba commit? Bugfixes are needed on a version that still supports Julia 0.4.

**Comment by jrevels** [comment]
Done.

**Pull request merged**